api/opencl_runtime_layer.asciidoc

```diff
@@ -16589,6 +16589,39 @@ device occupancy high by avoiding blocking in host code.
 
 [open,refpage='clUpdateMutableCommandsKHR',desc='Modify configuration of mutable-command handles to update behavior for future enqueues',type='protos']
 --
+
+==== Update to Command-Buffer Lifecycle
+
+If this extention is supported, the command-buffer lifecycle is extended to
+include a new state named **Finalized**.
```

Comment:
We should define a `clEnqueueCommandBufferKHR` error if a user tries to enqueue a command-buffer in the Finalized state without all it's arguments set.